### ISSUE TITLE
vue-cli 3.11.0 & vuepress 1.0.3 cause error

### ISSUE THREAD

**Issue by novbing** [vuejs/vue-cli#4539]
### Version
3.11.0

### Reproduction link
[https://github.com/novbing/issue__vue-cli__vuepress](https://github.com/novbing/issue__vue-cli__vuepress)

### Environment info
```
System:
    OS: macOS 10.14.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  Binaries:
    Node: 10.16.2 - /usr/local/bin/node
    Yarn: Not Found
    npm: 6.10.3 - /usr/local/bin/npm
  Browsers:
    Chrome: Not Found
    Firefox: 63.0.3
    Safari: 12.1.2
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0 
    @vue/babel-plugin-transform-vue-jsx:  1.0.0 
    @vue/babel-preset-app:  3.11.0 
    @vue/babel-preset-jsx:  1.1.0 
    @vue/babel-sugar-functional-vue:  1.0.0 
    @vue/babel-sugar-inject-h:  1.0.0 
    @vue/babel-sugar-v-model:  1.0.0 
    @vue/babel-sugar-v-on:  1.1.0 
    @vue/cli-overlay:  3.11.0 
    @vue/cli-plugin-babel: ^3.11.0 => 3.11.0 
    @vue/cli-plugin-eslint: ^3.11.0 => 3.11.0 
    @vue/cli-service: ^3.11.0 => 3.11.0 
    @vue/cli-shared-utils:  3.11.0 
    @vue/component-compiler-utils:  3.0.0 
    @vue/eslint-config-prettier: ^5.0.0 => 5.0.0 
    @vue/preload-webpack-plugin:  1.1.1 
    @vue/web-component-wrapper:  1.2.0 
    eslint-plugin-vue: ^5.0.0 => 5.2.3 (4.7.1)
    vue: ^2.6.10 => 2.6.10 
    vue-eslint-parser:  2.0.3 (5.0.0)
    vue-hot-reload-api:  2.3.3 
    vue-loader:  15.7.1 
    vue-router: ^3.0.3 => 3.1.3 
    vue-server-renderer:  2.6.10 
    vue-style-loader:  4.1.2 
    vue-template-compiler: ^2.6.10 => 2.6.10 
    vue-template-es2015-compiler:  1.9.1 
    vuepress: ^1.0.3 => 1.0.3 
    vuepress-html-webpack-plugin:  3.2.0 
    vuepress-plugin-container:  2.0.2 
    vuex: ^3.0.1 => 3.1.1 
  npmGlobalPackages:
    @vue/cli: 3.11.0
```

### Steps to reproduce
npm install
npm run docs:dev

### What is expected?
Page OK

### What is actually happening?
Error in console:

[Vue warn]: Failed to resolve async component: function Layout() {
    return __webpack_require__.e(/*! import() */ 0).then(__webpack_require__.bind(null, /*! ./node_modules/@vuepress/theme-default/layouts/Layout.vue */ "./node_modules/@vuepress/theme-default/layouts/Layout.vue"));
  }
Reason: TypeError: Cannot assign to read only property 'exports' of object '#<Object>'

**Comment by novbing** [comment]
same as [VuePress cannot coexist with Vue CLI generated app? #1807](https://github.com/vuejs/vuepress/issues/1807)

**Comment by LinusBorg** [comment]
Solved in vuepress 1.0.4
